.github/issue_template.md

```diff
@@ -0,0 +1,16 @@
+# Environment
+
+Please specify any relevant setup details. For example host OS, natively run node or docker, etc.
```

Comment:
Can we have these as comments?

Reply:
> do you mean quotes? 

like that?

Reply:
```
<!---
Description
-->
```

Currently, users will have to manually delete these so they don't show up in PR. Including them as comments means users can see them when opening a PR but they don't have to manually delete them.  It's just makes it easier.

Reply:
Ended up using html comments: https://github.com/fleek-network/ursa/pull/265/commits/4b57fb05dd1540da14921b36ca67a13a72bccdff